### ISSUE TITLE
error crear respositorio

### ISSUE THREAD

**Issue by 4gmz** [codesandbox/codesandbox-client#4431]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 83.0.4103 | Linux x86_64           | b29bb2b7f          |

  **Route:**
  https://codesandbox.io/s/github/4gmz/vue-storage/tree/master/

  **Error Code:**
27b2ccc60b99410d971b1c0d7a0d21bf

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
  Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at ys (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1755435)
    at ms (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1753436)
    at ys (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1755172)
    at yl (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1774437)
    at https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1795969
    at qo (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1720342)
    at bl (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1772322)
    at il (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1762649)
    at https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1720633
    at https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1795969
    at qo (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1720342)
    at Yo (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1720578)
    at $o (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1720513)
    at ce (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1789924)
    at On (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1704471)
    at Mn (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1705428)
    at Tn (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1704716)
    at https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1795969
    at jn (https://codesandbox.io/static/js/common.45abdf43d.chunk.js?_sw-precache=b88b58add97b52853f723473444a8de3:1:1704192)
  
    in elements__Container
    in div
    in hc
    in div
    in Styled(styled.div)
    in li
    in Entry___StyledListAction
    in div
    in n
    in div
    in Unknown
    in DragSource(Component)
    in div
    in elements__EntryContainer
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in tl
    in div
    in elements__Opener
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in tl
    in div
    in elements__Opener
    in div
    in Unknown
    in DropTarget(Component)
    in div
    in Styled(styled.div)
    in section
    in Styled(styled.div)
    in Unknown
    in Files___StyledCollapsible
    in yl
    in xl
    in github-summary
    in div
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in c
    in t
  ```

  </details>

**Comment by SaraVieira** [comment]
👋

Puedes explicar mejor qué hacía cuando sucedió esto?

 puedo abrir el repositorio ahora

**Comment by SaraVieira** [comment]
closed by #4571
